config/prow/autobump-config/prow-job-autobump-config.yaml

```diff
@@ -12,6 +12,8 @@ headBranchName: "prowjobs-autobump"
 upstreamURLBase: "https://raw.githubusercontent.com/kubernetes/test-infra/master"
 includedConfigPaths:
   - "config/jobs"
+excludedConfigPaths:
```

Comment:
Could you add a TODO here to remove this when things settle down?

Reply:
Done :+1: